ras/ras_ppcutils.py

```diff
@@ -428,6 +433,33 @@ def test_lparstat(self):
                 self.log.info("%s command passed" % cmd)
                 self.fail("lparstat: Expected failure, %s command exeucted \
                           successfully." % cmd)
+        output = process.system_output("lparstat -E 1 1").decode("utf-8")
+        input_string = ''
+        for line in output.splitlines():
+            if 'GHz' in line:
+                input_string = line
```

Comment:
cannot I use line directly, any need to assign to a new param ?

Reply:
yes, we can use the line directly .. Changed the code and re-pushed it again